test/functional/rpc_rawtransaction.py

```diff
@@ -260,71 +256,49 @@ def createrawtransaction_tests(self):
 
     def sendrawtransaction_tests(self):
         self.log.info("Test sendrawtransaction with missing input")
-        inputs = [{'txid': TXID, 'vout': 1}]  # won't exist
-        outputs = {self.nodes[0].getnewaddress(): 4.998}
-        rawtx = self.nodes[2].createrawtransaction(inputs, outputs)
-        rawtx = self.nodes[2].signrawtransactionwithwallet(rawtx)
```

Comment:
I think you can keep this test as-is and just remove this line and replace the call to getnewaddress

Reply:
Right, thanks!